Sources/Hypertext.swift

```diff
@@ -6,18 +6,23 @@
 //  Copyright © 2016 Sahand Nayebaziz. All rights reserved.
 //
 
-public protocol Renderable {
-    func render() -> String
-    func render(startingWithSpacesCount: Int) -> String
```

Comment:
I'm confused. What are you asking for that isn't implemented in this PR? I understand that you like the idea of having a simple `render()` function available that renders minimized, because that is the most common use case probably.

But you're talking about it as if this PR doesn't offer that, but it DOES offer that:

![screen shot 2016-11-08 at 16 27 41](https://cloud.githubusercontent.com/assets/1267282/20105255/365bbaa4-a5d1-11e6-8cc2-b18be5c721b7.png)